lib/inets/src/http_client/httpc_response.erl

```diff
@@ -124,15 +124,17 @@ result(Response = {{_, Code, _}, _, _},
 result(Response = {{_, Code, _}, _, _}, 
        Request = #request{settings =
               #http_options{autoredirect = true},
-              method = post}) when (Code =:= 307) ->
+              method = post}) when (Code =:= 307) orelse
+                           (Code =:= 308) ->
     redirect(Response, Request);
 result(Response = {{_, Code, _}, _, _}, 
        Request = #request{settings = 
 			  #http_options{autoredirect = true},
 			  method = Method}) when (Code =:= 301) orelse
 					       (Code =:= 302) orelse
 					       (Code =:= 303) orelse
-					       (Code =:= 307) ->
+					       (Code =:= 307) orelse
+                           (Code =:= 308) ->
```

Comment:
white at it, could you reformat that code block and make nesting little more consistent?